### ISSUE TITLE
Integration with existing givelight website

### ISSUE THREAD

**Issue by naderchehab** [Opportunity-Hack-San-Jose-2017/Team3#25]
This is a big task, I think the first step would be to gather more concrete steps we can take.

**Comment by JMStudiosJoe** [comment]
Yes we are integrated to their staging site which just redirects to our heroku app. We would need a way for the volunteer to get back to the next steps on the givelight.org site after they signup so probably just a button that links back to givelight for now?

**Comment by naderchehab** [comment]
Ok makes sense. Still it would be good if we could style our app to look the same as the existing site so people don't think like they left the website.
